swagger/openemr-api.yaml

```diff
@@ -6884,6 +7052,8 @@ components:
             user/surgery.write: 'Write surgeries the user has access to (api:oemr)'
             user/vital.read: 'Read vitals the user has access to (api:oemr)'
             user/vital.write: 'Write vitals the user has access to (api:oemr)'
+            user/transaction.read: ' (api:oemr)'
```

Comment:
@adunsulag , this file is built from _rest_routes.inc.php , so this will be fixed automagically after _rest_routes.inc.php is fixed and this file is rebuilt